bika/lims/locales/es/LC_MESSAGES/bika.po

```diff
@@ -7490,11 +7524,11 @@ msgstr "Dirección de correo electrónico de la entidad acreditadora"
 
 #: bika/lims/content/supplier.py:37
 msgid "Website."
-msgstr ""
+msgstr "Dirección web."
 
 #: bika/lims/browser/dashboard/templates/dashboard.pt:555
 msgid "Weekly"
-msgstr ""
+msgstr "Semana"
```

Comment:
@xispa I don't know the context here but, shouldn't weekly be translated to _Semanal_ ?